subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -144,6 +145,16 @@ public void setConsoleOutput(ConsoleOutput consoleOutput) {
         loggingConfiguration.setConsoleOutput(consoleOutput);
     }
 
+    @Override
+    public Set<WarningType> getWarningTypes() {
```

Comment:
:x: This should also be reflected in the methods `prepareNewInstance` and `toString`.

Reply:
Nice catch. Done.